src/instructlab/model/test.py

```diff
@@ -60,6 +60,11 @@
     "--model-family",
     help="Force model family to use when picking a generation template",
 )
+@click.option(
+    "--lora-enabled",
+    is_flag = True,
+    help="Enable or disable LORA during training"
```

Comment:
once this is an option in the configuration itself (as suggested in https://github.com/instructlab/instructlab/pull/2320#issuecomment-2379341121), please remove the `help` line from here, the help text is extracted directly from the description added in the config